doc/nrf/libraries/modem/lte_lc.rst

```diff
@@ -185,7 +185,7 @@ You can set the timer values requested by the modem using the following options:
 Connection pre-evaluation
 =========================
 
-Modem firmware version 1.3.0 and higher supports connection a pre-evaluation feature that allows the application to get information about a cell that is likely to be used for an RRC connection.
+Modem firmware version 1.3.0 and higher supports a connection pre-evaluation feature that allows the application to get information about a cell that is likely to be used for an RRC connection.
```

Comment:
```suggestion
Modem firmware versions 1.3.0 and higher support a connection pre-evaluation feature that allows the application to get information about a cell that is likely to be used for an RRC connection.
```

Reply:
@peknis, should I do this change? Few places in sdk-nrf with "version*higher" does not have "versions" but "version".

Reply:
I believe Mirko's suggestion works well, because this suggests that all versions starting from 1.3.0 support this feature.